README.md

```diff
@@ -81,7 +81,9 @@ require 'pry'
 
 # Install Puppet on all hosts
 hosts.each do |host|
-  on host, install_puppet
+  # Select the correct method below.
+  on host, install_puppet  # this is for Puppet 3
+  install_puppet_agent_on(host, :version => '1.5.2') # this is for Puppet 4
```

Comment:
ps - Doesn't the code say that install_puppet is deprecated in favor of `install_puppet_on()`? 
https://github.com/puppetlabs/beaker/blob/0e13b888970aae8fb82343078ce14eef69bb236c/lib/beaker/dsl/install_utils/foss_utils.rb#L204